README.md

```diff
@@ -55,22 +55,22 @@ $ npm install -g @angular/cli@latest
 * **Step 2:** Then cd into that cloned folder
 * **Step 3:** Then execute the command: ```$ npm install```
 * **Step 4:** Deploy locally by running this:```$ ng serve```
-* **Step 5:** Go to localhost:4200 where the application will be running locally.
+* **Step 5:** Go to [localhost:4200](http://localhost:4200) where the application will be running locally.
 
 ### For deploying with [Github Pages](https://pages.github.com/):
 With these very simple steps you can have susper deployed:
 
 * **Step 1:** Fork susper repository and clone it to your desktop.
 * **Step 2:** Then checkout to your master branch `$ git checkout master`
-* **Step 3:** Deploy running this: ```ng build``` and then ```npm run deploy```
+* **Step 3:** Deploy running this: ```$ ng build``` and then ```$ npm run deploy```
```

Comment:
No need to have `$`. Simple `ng build` and `npm run deploy` commands can be understood.